tests/runner.test.ts

```diff
@@ -181,6 +194,31 @@ describe('Runner', () => {
         new Error(`Invalid fiddle: "'invalid-fiddle'"`),
       );
     });
+
+    it('spawns a subprocess with ASAR path when runFromAsar is true', async () => {
+      const runner = await createFakeRunner({});
+      (child_process.spawn as jest.Mock).mockReturnValueOnce(mockSubprocess);
```

Comment:
```suggestion
      vi.mocked(child_process.spawn).mockReturnValueOnce(
        mockSubprocess as unknown as child_process.ChildProcess,
      );
```

Match other tests, and this repo does not use Jest.

Reply:
But the `package.json` includes Jest as a devDependency (not Vitest), and all existing tests use Jest syntax. e.g.,  https://github.com/electron/fiddle-core/blob/ad342621e6ff425d0164467b4c1c00339b95bf09/tests/runner.test.ts#L178 Am I missing something?

Reply:
@AlokYadavCodes, whoops, you're right! Apologies, we're in the middle of migrating a bunch of repositories from Jest to Vitest as part of an upgrade and my local checkout was a on a branch with that change when I glanced at the tests.

So you're right, no change needed here.